systemd/init

```diff
@@ -0,0 +1,49 @@
+#!/bin/busybox sh
+
+# Init for Wolfi OS
+# - Environment variables:
+#     VERBOSE=1 for debugging
+#     CMDLINE=/path/to/file to override /proc/cmdline, useful for testing
+# - Mount /proc, /sys, /dev if not already mounted
+# - Allow for override of path to /init with init=/path/to/init on kernel /proc/cmdline
+# - Allow for uuid or device path for root filesystem with root= on kernel /proc/cmdline
+# - Mount root, if specified and found, switch_root to it, and run systemd as init
+# - Otherwise, try to run systemd as init in the initrd
+
+# Verbose debugging
+info() { [ "$VERBOSE" = "1" ] && printf "INFO: $@\n" 1>&2; }
```

Comment:
you probably mean
```
printf "INFO: %s\n" "$*" 1>&1
```

at least i think thats how you're using it.  ie, you aren't intending to expose printf's formatting to callers.

Reply:
Sure, that works.